### DISCUSSION TITLE
Certificate Registry Integration with RC for download and search certificate

### DISCUSSION THREAD

**Discussion by sravani-64** [Sunbird-Ed/Community#583] [Installation] [answered]
Hi team,

We have setup RC jobs on release-4.10.0 
To integrate Certificate registry with RC to download and search certificate - https://project-sunbird.atlassian.net/browse/SB-29068 we were not able to find the integration procedure in this document. Can someone help us here?
Thank you
CC @Krishnaj20 @gandham-santhosh

**Comment by rajeevsathish** [comment, marked as answer]
@sravani-64 you can find the implementation in this following code
https://github.com/Sunbird-Ed/SunbirdEd-portal/blob/release-4.10.0/src/app/client/src/app/plugins/profile/components/profile-page/profile-page.component.ts#L344
cc: @rajeshkumaravel @vinukumar-vs @chitranshu-keshav
